python/pyspark/sql/functions.py

```diff
@@ -312,7 +338,7 @@ def covar_samp(col1, col2):
 
 @since(1.3)
 def countDistinct(col, *cols):
-    """Returns a new :class:`Column` for distinct count of ``col`` or ``cols``.
+    """Returns a new :class:`Column` for distinct count of `col` or `cols`.
```

Comment:
I guess we did not decide which one of `` `...` `` or ``` ``...`` ``` is more preferred and right. Let's avoid sweeping them. Adding new one is fine (and I usually follow majority in this case) but changing the existing one might be not if we don't know which one is not preferred. We might happen to change it forward and backward in time.